### ISSUE TITLE
Add Raw Headers

### ISSUE THREAD

**Issue by pafuent** [esphen/RESTED#45]
It would be nice if you can add headers to a request in raw format, like Postman in Chrome.

**Comment by esphen** [comment]
Yeah, not a bad idea. It should not be that hard either. Simply parse every line as `key:value` and put into request headers object. But as I mentioned earlier, I'm rather busy as of late, so I'll have a look at it when I have time :)
